### ISSUE TITLE
Legacy .plx (PlexonIO) read error

### ISSUE THREAD

**Issue by MisterVulcan** [NeuralEnsemble/python-neo#292]
I'm using an old version of Plexon's Offline Sorter (2.8.8) and Sorting Client (1.21) software which saves with the old .plx format. Python-neo version 0.4.1. Below is the typical use case.

```
In [31]: reader = PlexonIO(filename='03-07_1650.plx')

In [32]: blks = reader.read(cascade=True, lazy=False)
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-32-1dd5e1af6904> in <module>()
----> 1 blks = reader.read(cascade=True, lazy=False)

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/neo/io/baseio.pyc in read(self, lazy, cascade, **kargs)
    119             if not cascade:
    120                 return bl
--> 121             seg = self.read_segment(lazy=lazy, cascade=cascade,  **kargs)
    122             bl.segments.append(seg)
    123             bl.create_many_to_one_relationship()

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/neo/io/plexonio.py in read_segment(self, lazy, cascade, load_spike_waveform)
    161
    162             if dataBlockHeader['Type'] == 1:
--> 163                 nb_spikes[chan,unit] +=1
    164                 wf_sizes[chan,unit,:] = [n1,n2]
    165                 fid.seek(n1*n2*2,1)

IndexError: index 1 is out of bounds for axis 1 with size 1
```
Note that this _sort of_ used to work with an earlier version of neo, and only when I was working through the OpenElectrography interface and not a python prompt. Now it's just broke. Any tips?

**Comment by samuelgarcia** [comment]
@MisterVulcan: sorry for very long delay. Have you tested version neo version 0.5.1  or curent master branch ?

**Comment by MisterVulcan** [comment]
Sorry for the long delay in my reply, haha. I installed `neo-0.6.0`, and it appears to work fine compared to the last time I tried. Right now my workflow consists of calling an octave function from within python to get continuous channels, so I'll have to make a few changes to ensure everything works just right. While I'm here, does `io.PlexonIO` contain any functions for extracting tracking data? All of the official and unofficial libraries I've found only contain functions for getting spiking/analog channel data from `.plx` files, even though I _know_ the files contain coordinate data for Plexon's tracking system. Right now the only way to get that data is to open NeuroExplorer, so an in-library solution would be most welcome. Thanks!
